### ISSUE TITLE
Fix FP S1128 ('unused-import'): Generalize imports matching kebab case references

### ISSUE THREAD

**Issue by yassin-kammoun-sonarsource** [SonarSource/SonarJS#3125]
S1128 raises a false positive for the following Vue snippet because the implementation only considers the first two words of a component named with kebab case.

```vue
<script setup>
  import FooBarBaz from './FooBarBaz.vue' // FP: `FooBarBaz` is wrongly raised as being unused
</script>      
<template>
  <foo-bar-baz />
</template>
```
